.github/workflows/auto-assign-to-project.yml

```diff
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request_target:
+    types: [opened, labeled]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.AUTO_ASSIGN_WORKFLOW_TOKEN }}
```

Comment:
@ericcornelissen Would you mind to set again the `AUTO_ASSIGN_WORKFLOW_TOKEN` as an organization secret, if is not enabled yet?

Reply:
Done :+1: